examples/example_socks4.py

```diff
@@ -0,0 +1,43 @@
+import socket
+import socksio
+from socksio import socks4
+
+MAX_BUFFERSIZE = 1024
+
+
+def send_data(sock, data):
+    print("sending:", data)
+    sock.sendall(data)
+
+
+def receive_data(sock):
+    data = sock.recv(1024)
+    print("received:", data)
+    return data
+
+
+def main():
+    # Assuming a running SOCKS4 proxy running in localhost:8080
+    sock = socket.create_connection(("localhost", 8080))
+    conn = socksio.socks4.SOCKS4Connection(user_id=b"foo")
```

Comment:
Could we remove the import to `socksio` and use `socks4` or `socks5` in each example script?

Reply:
Yeah, absolutely.